src/metapath-results.js

```diff
@@ -623,14 +639,15 @@ function makeMetapathsTable(metapaths) {
     metapath_name: 2,
     metapath_abbreviation: 3,
     path_count: 4,
-    p_value: 5,
-    dwpc: 6,
-    dgp_source_degree: 7,
-    dgp_target_degree: 8,
-    dgp_n_dwpcs: 9,
-    dgp_n_nonzero_dwpcs: 10,
-    dgp_nonzero_mean: 11,
-    dgp_nonzero_sd: 12
+    adjusted_p_value: 5,
+    p_value: 6,
+    dwpc: 7,
+    dgp_source_degree: 8,
+    dgp_target_degree: 9,
```

Comment:
I think @zietzm suggested that the Null distribution bar should not include `dgp_source_degree` and `dgp_target_degree`. However, I think it actually makes sense that it does, because while they apply to both the DWPC and null DWPC distribution, they are mostly important in the context of the null distribution.